main.go

```diff
@@ -28,6 +28,10 @@ import (
 	"go.bug.st/serial"
 )
 
+var (
+	gitSha1 string
```

Comment:
Can you add a comment here explaining that this variable is set to the git SHA1 hash with a build flag?

Reply:
Added a comment.
I've added a link to stackoverflow, but it may not be necessary.